mobify/platform-scaffold
# [Integration Manager] Plp demandware connector

This PR adds commands to the demandware connector that fetch content for the PLP. It also updates the PDP to account for product variation selection.


 **JIRA**: https://mobify.atlassian.net/browse/WEB-1255
 **Linked PRs**: Blocked by #438 

## Changes
- Fetch and parse content for the PLP in the demandware connector
- Parse and store product variation data on the PDP
- Render product variations if they exist
- Fetch and render new product data when variations are selected
- Add product variation data to the add to cart request

## How to test-drive this PR
- Set up tag injector for the demandware sandbox (http://mobify-tech-prtnr-na03-dw.demandware.net/on/demandware.store/Sites-2017refresh-Site/default/Home-Show)
- Run `npm run dev`
- Preview [the demandware site](https://preview.mobify.com/?url=http%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to http://mobify-tech-prtnr-na03-dw.demandware.net/s/2017refresh/womens-clothing
- The page should work as expected
- Navigate to a PDP from the page
- Select product variations and add to cart
